### PR TITLE
[csrng/doc] Adapt the command status response documentation

### PR DESCRIPTION
**Pull request by h-filali**
As discussed in #24193 this is the PR to adapt the documentation.

### CODE REVIEW

File: hw/ip/csrng/doc/theory_of_operation.md [hunk 1]
```diff
@@ -284,8 +284,8 @@ The actions performed by each command, as well as which flags are supported, are
 #### Command Response
 
 Once a command has been completed, successfully or unsuccessfully, the CSRNG responds with a single cycle pulse on the `csrng_rsp_ack` signal associated with the same application interface port.
-If the command is successful, the `csrng_rsp_sts` signal will indicate the value 0 (`CSRNG_OK`) in the same cycle.
-Otherwise the application will receive the value 1 (`CSRNG_ERROR`) on the `csrng_rsp_sts` signal.
+If the command is successful, the `csrng_rsp_sts` signal will indicate the value 0 (`SUCCESS`) in the same cycle.
+Otherwise the application will receive the an error value on the `csrng_rsp_sts` signal as described in [`SW_CMD_STS.CMD_STS`](registers.md#sw_cmd_sts).
```
Comment (vogelpi) [inline hw/ip/csrng/doc/theory_of_operation.md:288]: the an -> an

**Review by vogelpi — APPROVED** [review]
This looks good - except for one typo. Thanks for doing this @h-filali !

**Comment by vogelpi** [comment]
CHANGE AUTHORIZED: hw/ip/csrng/data/csrng.hjson

This PR touches the hjson for documentation purposes only. The generated RTL code remains unchanged. This is okay.

File: hw/ip/csrng/doc/theory_of_operation.md (hunk 1 above)
Comment (h-filali) [reply hw/ip/csrng/doc/theory_of_operation.md]: Thanks, good catch! I changed it now.

**Comment by andreaskurth** [comment]
CHANGE AUTHORIZED: hw/ip/csrng/data/csrng.hjson

**Pull request merged**